### ISSUE TITLE
Add the ability to disable asserts

### ISSUE THREAD

**Issue by bilel-noomene** [1up-lab/OneupFlysystemBundle#139]
I don't see any method to configure the disable_asserts parameter. The only parameter that can be passed to the Filesystem configuration is visibility.
```
        $options = [];
        if (array_key_exists('visibility', $config)) {
            $options['visibility'] = $config['visibility'];
        }

        $container
            ->setDefinition($id, new DefinitionDecorator('oneup_flysystem.filesystem'))
            ->replaceArgument(0, new Reference($cache ? $adapter.'_cached' : $adapter))
            ->replaceArgument(1, $options)
            ->addTag('oneup_flysystem.filesystem', $tagParams);
```
Thanks.

**Comment by bytehead** [comment]
See PR #140.

**Comment by bilel-noomene** [comment]
thanks

**Comment by bytehead** [comment]
Implemented via d6f8821ccf173f171dd76258bda3abafd218f0af. Check new release `1.13.0`.
